test/tests/utils/spline_interpolation/tests

```diff
@@ -9,5 +9,20 @@
     input = 'bicubic_spline_interpolation.i'
     exodiff = 'bicubic_spline_interpolation_out.e'
     custom_cmp = 'bicubic_spline_interpolation.cmp'
+    group = 'x'
```

Comment:
What's the purpose of this group label? If it's useful, perhaps a more descriptive name would be better?

Reply:
If I don't here anything soon, I'm going to push deletion of this group parameter and then merge because this should go in...

Reply:
whoops, I was using that group to run tests and only run that test